src/main/java/teammates/common/util/Const.java

```diff
@@ -870,18 +870,45 @@
+                                        + "Google account is not known to TEAMMATES." 
+                                        + "</div>"
+                                        + "</div>"
+                                        + "<br>"
+                                        + "<div style=\"text-align:left; \">"
+                                        + "<br>"
+                                        + "To access a course on TEAMMATES, first you need to wait till an "
+                                        + "instructor adds you to that course and TEAMMATES sends you "
+                                        + "instructions on how to access that particular course in TEAMMATES."
+                                        + "<br><br>If you 'joined' the course in TEAMMATES using a Google "
+                                        + "account before,but cannot login anymore, these are the possible reasons:"
+                                        + "<ol>"
+                                        + "<li>"
+                                        + "You used a different Google account to access TEAMMATES in the "
+                                        + "past. In that case, you need to use the same Google account to access "
+                                        + "TEAMMATES again. Logout and re-login using the other Google account. "
+                                        + "If you don't remember which Google account you used previously, email "
+                                        + "us from the same email account to which you receive TEAMMATES emails."
+                                        + "</li>"
+                                        + "<li>"
+                                        + "You changed the primary email from a non-Gmail address to a "
+                                        + "Gmail address recently. In that case, "
+                                        + "<a href=\"https://teammatesv4.appspot.com/contact.html\">"
+                                        + "email us "
+                                        + "</a>"
+                                        + "so that we can reconfigure your account to use the new Gmail address."
+                                        + "</li>"
+                                        + "<li>"
+                                        + "You joined this course just a few seconds ago and your data "
+                                        + "may be still in the process of propagating through our servers. "
+                                        + "In that case, please click on the "
+                                        + "<a href=" + ActionURIs.STUDENT_HOME_PAGE + ">Home</a> link above in "
+                                        + "a few minutes."
+                                        + "</li>"
+                                        + "</ol>"
+                                        + "</div>";
```

Comment:
Also, as @wkurniawan07  mentioned, do indent the tags according to their nesting to make it more readable.
E.g.
```
public static final String STUDENT_FIRST_TIME =
          "<div>"
            + "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor "
            + "incididunt ut labore et dolore magna aliqua."
        + "</div>";
```